frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -152,6 +154,14 @@ export default function AddUser(): JSX.Element {
       return;
     }
 
+    // Check that college is selected
+    if (!member.college) {
+      Emitters.generalError.emit({
+        headerMsg: 'College not selected!',
+        contentMsg: 'Please select college from the dropdown.'
+      });
```

Comment:
Make sure to `return` here, right now it still lets me submit even though I'm missing this field. That said we will likely need to take this away for now since no one has their college populated yet. So it's going to show an error every time an admin tries to save the user, if the college isn't populated yet

Reply:
also, if we're going to require `Semester Joined`, we should require it here too so admins don't forget. In fact, if you don't, members aren't going to be able to save profiles because the semester joined field won't be populated (and they can't fix it because the field is disabled on the member side)